rtos/rtx2/TARGET_CORTEX_M/RTX_Config.c

```diff
@@ -139,6 +139,9 @@
 //     <0=> Unprivileged mode 
 //     <1=> Privileged mode
 //   <i> Default: Privileged mode
+#if defined(FEATURE_UVISOR) && defined(TARGET_UVISOR_SUPPORTED)
+#define OS_PRIVILEGE_MODE           0
```

Comment:
That should be done in this file https://github.com/ARMmbed/mbed-os/blob/feature_cmsis5/rtos/rtx2/mbed_rtx_conf.h